roles/kubernetes/node/templates/kubelet-container.j2

```diff
@@ -14,6 +14,9 @@
   {% for dir in ssl_ca_dirs -%}
   -v {{ dir }}:{{ dir }}:ro \
   {% endfor -%}
+  {% if kubelet_load_modules is defined and kubelet_load_modules == true -%}
```

Comment:
add a default in roles/kubernetes/node/defaults and to avoid always having to write "VARNAME is defined" checks.
Also you can just do `{% if kubelet_load_modules %}`